flask-discord/flaskd/schema.sql

```diff
@@ -4,7 +4,8 @@ DROP TABLE IF EXISTS siteViews;
 
 CREATE TABLE user (
 	id INTEGER PRIMARY KEY AUTOINCREMENT,
-	username TEXT UNIQUE NOT NULL,
+	email TEXT UNIQUE NOT NULL,
```

Comment:
Great, we added functionality to have any username irrespective of having to have it unique